Foundry V10 Data Object Error

Hi there! Love the idea of this module for a little old school adventure game I'm running. I don't know if you're still maintaining this, but I got this error message in V10:

```
commons.js:1703 Error: You are accessing the DrawingDocument#data object which is no longer used. Since V10 the Document class and its contained DataModel are merged into a combined data structure. You should now reference keys which were previously contained within the data object directly.
    at logCompatibilityWarning (commons.js:1692)
    at Function._logV10CompatibilityWarning (commons.js:6642)
    at DrawingDocument.get data [as data] (commons.js:6542)
    at PerspectiveTokenScaler.js:6
    at Map.filter (commons.js:3244)
    at Object.handleUpdate [as fn] (PerspectiveTokenScaler.js:6)
    at Function.#call (foundry.js:724)
    at Function.call (foundry.js:706)
    at ClientDatabaseBackend._preUpdateDocumentArray (foundry.js:12736)
    at async ClientDatabaseBackend._updateEmbeddedDocuments (foundry.js:12661)
    at async Function.updateDocuments (commons.js:6153)

```

--- a/module.json
+++ b/module.json
@@ -5,8 +5,8 @@
     "version": "1.0.0",
     "library": "false",
     "manifestPlusVersion": "1.0.0",
-    "minimumCoreVersion": "0.8.9",
-    "compatibleCoreVersion": "0.8.9",
+    "minimumCoreVersion": "10.260",
+    "compatibleCoreVersion": "11.315",
     "authors": [
         {
             "name": "Tobias Franz",

--- a/scripts/PerspectiveTokenScaler.js
+++ b/scripts/PerspectiveTokenScaler.js
@@ -3,8 +3,8 @@ export class PerspectiveTokenScaler {
     static handleUpdate(token, change) {
         if (token instanceof TokenDocument && (change.x || change.y)) {
             let currentScene = game.scenes.viewed
-            let starterData = currentScene.drawings.filter(d => d.data.text === 'scaler_start')?.[0]?.data
-            let limiterData = currentScene.drawings.filter(d => d.data.text === 'scaler_limit')?.[0]?.data
+            let starterData = currentScene.drawings.filter(d => d.text === 'scaler_start')?.[0]
+            let limiterData = currentScene.drawings.filter(d => d.text === 'scaler_limit')?.[0]
 
             if (typeof starterData !== 'undefined' && typeof limiterData !== 'undefined') {
                 let xDiffMax = limiterData.x - starterData.x

--- a/scripts/PerspectiveTokenScaler.js
+++ b/scripts/PerspectiveTokenScaler.js
@@ -15,7 +15,11 @@ export class PerspectiveTokenScaler {
                 let yDiffCurrent = limiterData.y - (change.y ? change.y : 0)
                 let distance = Math.sqrt(xDiffCurrent * xDiffCurrent + yDiffCurrent * yDiffCurrent)
                 let factor = distance > maxDistance ? 1 : distance / maxDistance
-                change.scale = factor < 0.25 ? 0.25 : factor
+                
+                change.texture = {
+                    'scaleX' : factor < 0.25 ? 0.25 : factor,
+                    'scaleY' : factor < 0.25 ? 0.25 : factor
+                }
             }
         }
     }

